docs/developer-docs/integrations/sns/launching/launch-steps-1proposal.md

```diff
@@ -42,6 +42,14 @@ They can do so by running the following command:
 dfx sns prepare-canisters add-nns-root $CANISTER_ID
 ```
 
+They also remove themselves from the list of principals that may change frontend canister content on their own.
+
+```bash
+dfx canister call $CANISTER_ID revoke_permission '(record {of_principal = principal "<developer principal"; permission = variant { Commit;};})'
+```
+
+If any other canisters in the dapp contain custom access control, these permission lists should also be prepared for decentralization.
```

Comment:
I would have started by the more general statement and then just name the asset canister as an example (made a proposal above before seing this). I am not sure if both is needed but would probably also be OK.